Backpack/FlightLeg/Classes/BPKFlightLeg.swift

```diff
@@ -146,15 +146,18 @@ public final class BPKFlightLeg: UIView {
         let verticalStackView = UIView()
         verticalStackView.translatesAutoresizingMaskIntoConstraints = false
         
-        carrierLogoIcon.iconName = carrierLogo
+        carrierLogoIcon.image = carrierLogo
         leadingStackView.addArrangedSubview(carrierLogoIcon)
         verticalStackView.addSubview(leadingStackView)
         
         NSLayoutConstraint.activate([
             leadingStackView.topAnchor.constraint(equalTo: verticalStackView.topAnchor, constant: BPKSpacingSm),
             leadingStackView.bottomAnchor.constraint(equalTo: verticalStackView.bottomAnchor),
             leadingStackView.leadingAnchor.constraint(equalTo: verticalStackView.leadingAnchor),
-            leadingStackView.trailingAnchor.constraint(equalTo: verticalStackView.trailingAnchor)
+            leadingStackView.trailingAnchor.constraint(equalTo: verticalStackView.trailingAnchor),
+            
+            carrierLogoIcon.widthAnchor.constraint(equalToConstant: 24),
+            carrierLogoIcon.heightAnchor.constraint(equalToConstant: 24)
```

Comment:
Can we use BPKSpacings for these 'magic' numbers. A constant defined at the top is also ok.